### PR TITLE
publish another beta

### PR DESCRIPTION
**Pull request by shiftkey**
Feel free to leave comments on the release notes, I'll follow up afterwards with edits.

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "1.3.0-beta2": [
+      "[Fixed] Diff gutter elements should be considered button elements when interacting - #5158",
+      "[Fixed] Add option to move repository to trash when removing from app - #2108. Thanks @say25!",
```
Comment (j-f1) [inline changelog.json]: IMO this should be a `[New]` or `[Added]`.

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "1.3.0-beta2": [
+      "[Fixed] Diff gutter elements should be considered button elements when interacting - #5158",
```
Comment (j-f1) [inline changelog.json]: `[Improved]`?

File: changelog.json (hunk 1 above)
Comment (shiftkey) [reply changelog.json]: :thumbsup: to `Added`

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "1.3.0-beta2": [
+      "[Added] Add option to move repository to trash when removing from app - #2108. Thanks @say25!",
```
Comment (iAmWillShepherd) [inline changelog.json]: 🔥 "Add"

**Pull request merged**